indigo/distribution.yaml

```diff
@@ -13335,6 +13335,22 @@ repositories:
       url: https://github.com/ros-simulation/stage_ros.git
       version: master
     status: maintained
+  static_tf:
+    doc:
+      type: git
+      url: https://github.com/DLu/static_tf.git
+      version: master
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/wu-robotics/static_tf_release.git
+      version: 0.0.1-1
+    source:
+      test_pull_requests: true
```

Comment:
Where are those instructions? I just followed the instructions for bloom as per usual. Not sure where that line came from.